marbot.yml

```diff
@@ -1407,6 +1413,42 @@ Resources:
       Targets:
       - Arn: !Ref Topic
         Id: marbot
+  ECSSpotFailureEvent:
+    Condition: ECSSpotNotificationsEnabled
+    DependsOn: TopicEndpointSubscription
+    Type: 'AWS::Events::Rule'
+    Properties:
+      Description: 'ECS Spot failure. (created by marbot)'
```

Comment:
ECS service scheduler was unable to acquire any Fargate Spot capacity. (created by marbot)

Reply:
✅